Makefile

```diff
@@ -140,10 +140,14 @@ test-image-all:
 # -----------
 
 .PHONY: test-e2e
-test-e2e: test-e2e-1.21
+test-e2e: test-e2e-1.23
 
 .PHONY: test-e2e-all
-test-e2e-all: test-e2e-1.22 test-e2e-1.21 test-e2e-1.20 test-e2e-1.19
+test-e2e-all: test-e2e-1.23 test-e2e-1.22 test-e2e-1.21 test-e2e-1.20 test-e2e-1.19
```

Comment:
I think we should follow Kubernetes support policy here and just support the 3 latest releases

Reply:
I agree, it seems that the test takes a long time now

Reply:
Done